first_diff.py

```diff
@@ -1,255 +1,58 @@
 #!/usr/bin/env python3
 
-import os.path
 import argparse
-from subprocess import check_call
-
-parser = argparse.ArgumentParser(
-    description="Find the first difference(s) between the built ROM and the base ROM."
-)
-parser.add_argument(
-    "-c",
-    "--count",
-    type=int,
-    default=5,
-    help="find up to this many instruction difference(s)",
-)
-parser.add_argument(
-    "-d",
-    "--diff",
-    dest="diff_args",
-    nargs="?",
-    action="store",
-    default=False,
-    const="prompt",
-    help="run diff.py on the result with the provided arguments"
-)
-parser.add_argument(
-    "-m", "--make", help="run make before finding difference(s)", action="store_true"
-)
-args = parser.parse_args()
-
-diff_count = args.count
-
-if args.make:
-    check_call(["make", "-j4", "COMPARE=0"])
-
-baseimg = f"baserom.z64"
-basemap = f"expected/build/z64.map"
-
-myimg = f"zelda_ocarina_mq_dbg.z64"
-mymap = f"build/z64.map"
-
-if not os.path.isfile(baseimg):
-    print(f"{baseimg} must exist.")
+from pathlib import Path
+
+try:
+    import rabbitizer
+except ImportError:
+    print("Missing dependency rabbitizer, install it with `python3 -m pip install rabbitizer>=1.0.0,<2.0.0`")
```

Comment:
```suggestion
    print("Missing dependency rabbitizer, install it with `python3 -m pip install 'rabbitizer>=1.0.0,<2.0.0'`")
```
and similar with the others (bash sees the <> as special shell characters)